src/main/java/roomescape/entities/Reservation.java

```diff
@@ -0,0 +1,76 @@
+package roomescape.entities;
+
+import jakarta.persistence.Entity;
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.GenerationType;
+import jakarta.persistence.Id;
+import jakarta.persistence.JoinColumn;
+import jakarta.persistence.OneToOne;
+import lombok.Builder;
+import roomescape.exceptions.ErrorCode;
+import roomescape.exceptions.RoomEscapeException;
+
+import java.time.LocalDate;
+import java.time.LocalDateTime;
+import java.time.LocalTime;
+
+@Builder
+@Entity
+public class Reservation {
+
+  @Id @GeneratedValue(strategy = GenerationType.IDENTITY)
+  private Long id;
+  private String name;
+  private String date;
+
+  @OneToOne
+  @JoinColumn(name = "reservation_time_id")
```

Comment:
schema.sql 파일을 고려했을 때 다음의 컬럼명이 좋을 것 같습니다!
```suggestion
  @JoinColumn(name = "time_id")
```